src/main/kotlin/io/emeraldpay/dshackle/rpc/NativeCall.kt

```diff
@@ -119,6 +119,9 @@ open class NativeCall(
                 result.setErrorMessage(error.message).setErrorCode(error.id)
             }
         } else {
+            if (it.result == null || it.result.isEmpty()) {
+                log.warn("Empty result on building response, method ${it.ctx?.payload?.method}, params ${it.ctx?.payload?.method}")
+            }
```

Comment:
`method ${it.ctx?.payload?.method}, params ${it.ctx?.payload?.method}" `
**copy/paste**

Reply:
thanks